agent/kubernetes/service.go

```diff
@@ -37,13 +38,17 @@ func (cs *CostService) processRawData(nodes []v1.Node, pods []v1.Pod) {
 	log.Info("processing raw data...")
 
 	for _, node := range nodes {
+		externalID := node.Spec.ExternalID
 		allocatableMemory := node.Status.Allocatable["memory"]
 		allocatableCPU := node.Status.Allocatable["cpu"]
 		capacityMemory := node.Status.Capacity["memory"]
 		capacityCPU := node.Status.Capacity["cpu"]
+		region := node.ObjectMeta.Labels["failure-domain.beta.kubernetes.io/region"]
```

Comment:
What is this?
"failure-domain.beta.kubernetes.io/region"

Reply:
This is the AWS region that is set as a label, for example:
```json
"labels": {
  "failure-domain.beta.kubernetes.io/region": "us-east-1",
  "beta.kubernetes.io/os": "linux",
  "failure-domain.beta.kubernetes.io/zone": "us-east-1e",
  "node-role.kubernetes.io/node": "",
  "zappi.it/role": "general",
  "beta.kubernetes.io/arch": "amd64",
  "kubernetes.io/hostname": "ip-10-83-115-12.ec2.internal",
  "kubernetes.io/role": "node",
  "beta.kubernetes.io/instance-type": "r4.xlarge"
}
```

Reply:
Ah, understood